test_scripts/Polices/Validation_of_PolicyTables/233_ATF_preloaded_pt_requesttype_parameters_all_are_invalid.lua

```diff
@@ -21,14 +21,33 @@ local commonSteps = require ('user_modules/shared_testcases/commonSteps')
 local commonPreconditions = require ('user_modules/shared_testcases/commonPreconditions')
 local testCasesForPolicySDLErrorsStops = require ('user_modules/shared_testcases/testCasesForPolicySDLErrorsStops')
 local testCasesForPolicyTable = require ('user_modules/shared_testcases/testCasesForPolicyTable')
+local sdl = require('modules/SDL')
+local testCasesForExternalUCS = require('user_modules/shared_testcases/testCasesForExternalUCS')
+local json = require("modules/json")
 
 --[[ General configuration parameters ]]
 config.defaultProtocolVersion = 2
 config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.ExitOnCrash = false
 commonSteps:DeleteLogsFileAndPolicyTable()
 
 --[[ Local Variables ]]
-local PRELOADED_PT_FILE_NAME = "sdl_preloaded_pt.json"
+local function update_preloaded_file()
```

Comment:
@dboltovskyi, yes it is done to update data only if exist, but in original preloaded_pt this element is not listed.